src/index.js

```diff
@@ -1,16 +1,18 @@
+const linguistData = require('linguist-languages/data/Solidity.json');
 const { handleComments, printComment } = require('./comments');
-
 const massageAstNode = require('./clean');
 const loc = require('./loc');
 const options = require('./options');
 const parse = require('./parser');
 const print = require('./printer');
 
 // https://prettier.io/docs/en/plugins.html#languages
+const { languageId, ...rest } = linguistData;
 const languages = [
   {
-    extensions: ['.sol'],
-    name: 'Solidity',
+    linguistLanguageId: languageId,
+    ...rest,
+    since: '0.0.0',
```

Comment:
is this required?

Reply:
it's optional but am unsure if it's prettier's version, or our plugin's version